docs/modules/operation/pages/flows/sentinel/message-broker/kafka.adoc

```diff
@@ -25,6 +25,7 @@ ssh -p 8301 admin@localhost
 config:edit org.opennms.sentinel.controller
```

Comment:
This file is missing a top level heading and is not part of the navigation.

Reply:
It is content in a tab (like we use for the different installation OSs), so doesn't get a top level heading and is not part of the nav.